indexing-hadoop/src/test/java/io/druid/indexer/updater/HadoopConverterJobTest.java

```diff
@@ -451,6 +452,7 @@ private static void corrupt(
   }
 
   @Test
+  @Ignore // This takes a long time due to retries
```

Comment:
Right now the retries are static in the class. To have them configurable is a big ask here.